modules/tensor_mechanics/src/actions/TensorMechanicsActionBase.C

```diff
@@ -18,38 +18,71 @@ const std::map<std::string, std::string>
-        {"vonmises", {"VonMisesStress", {"stress"}}},
-        {"effective", {"EffectiveStrain", {"plastic_strain", "creep_strain"}}},
-        {"hydrostatic", {"Hydrostatic", {"stress"}}},
-        {"l2norm", {"L2norm", {"stress", "strain"}}},
-        {"volumetric", {"VolumetricStrain", {"strain"}}},
-        {"firstinv", {"FirstInvariant", {"stress", "strain"}}},
-        {"secondinv", {"SecondInvariant", {"stress", "strain"}}},
-        {"thirdinv", {"ThirdInvariant", {"stress", "strain"}}},
-        {"triaxiality", {"TriaxialityStress", {"stress"}}},
-        {"maxshear", {"MaxShear", {"stress"}}},
-        {"intensity", {"StressIntensity", {"stress"}}},
-        {"max_principal", {"MaxPrincipal", {"stress"}}},
-        {"mid_principal", {"MidPrincipal", {"stress"}}},
-        {"min_principal", {"MinPrincipal", {"stress"}}}};
+        {"vonmises",
+         {"VonMisesStress",
+          {"stress", "strain", "elastic_strain", "plastic_strain", "creep_strain"}}},
+        {"effective",
+         {"EffectiveStrain",
+          {"stress", "strain", "elastic_strain", "plastic_strain", "creep_strain"}}},
+        {"hydrostatic",
+         {"Hydrostatic", {"stress", "strain", "elastic_strain", "plastic_strain", "creep_strain"}}},
+        {"l2norm",
+         {"L2norm", {"stress", "strain", "elastic_strain", "plastic_strain", "creep_strain"}}},
+        {"volumetric",
+         {"VolumetricStrain",
+          {"stress", "strain", "elastic_strain", "plastic_strain", "creep_strain"}}},
+        {"firstinv",
+         {"FirstInvariant",
+          {"stress", "strain", "elastic_strain", "plastic_strain", "creep_strain"}}},
+        {"secondinv",
+         {"SecondInvariant",
+          {"stress", "strain", "elastic_strain", "plastic_strain", "creep_strain"}}},
+        {"thirdinv",
+         {"ThirdInvariant",
+          {"stress", "strain", "elastic_strain", "plastic_strain", "creep_strain"}}},
+        {"triaxiality",
+         {"TriaxialityStress",
+          {"stress", "strain", "elastic_strain", "plastic_strain", "creep_strain"}}},
+        {"maxshear",
+         {"MaxShear", {"stress", "strain", "elastic_strain", "plastic_strain", "creep_strain"}}},
+        {"intensity",
+         {"StressIntensity",
+          {"stress", "strain", "elastic_strain", "plastic_strain", "creep_strain"}}},
+        {"max_principal",
+         {"MaxPrincipal",
+          {"stress", "strain", "elastic_strain", "plastic_strain", "creep_strain"}}},
+        {"mid_principal",
+         {"MidPrincipal",
+          {"stress", "strain", "elastic_strain", "plastic_strain", "creep_strain"}}},
+        {"min_principal",
+         {"MinPrincipal",
+          {"stress", "strain", "elastic_strain", "plastic_strain", "creep_strain"}}}};
 
 const std::map<std::string, std::pair<std::string, std::vector<std::string>>>
     TensorMechanicsActionBase::_rank_two_directional_component_table = {
-        {"directional", {"Direction", {"stress", "strain"}}}};
+        {"directional",
```

Comment:
Same deal with all of these. Let's revert them back for now, and potentially add some as we add tests.